src/Features/CSharp/CSharpFeaturesResources.resx

```diff
@@ -228,6 +228,9 @@
   <data name="Deprecated" xml:space="preserve">
     <value>deprecated</value>
```

Comment:
I can do that. It seems like there is an inconsistency with the casing in the VB and CS strings.

VB strings:

<data name="Deprecated" xml:space="preserve">
    <value>Deprecated</value>
  </data>
  <data name="Obsolete" xml:space="preserve">
    <value>Obsolete</value>
  </data>
  <data name="Extension" xml:space="preserve">
    <value>Extension</value>
  </data>
  <data name="Awaitable" xml:space="preserve">
    <value>Awaitable</value>
  </data>
  <data name="AwaitableExtension" xml:space="preserve">
    <value>Awaitable, Extension</value>
  </data>
  <data name="Await" xml:space="preserve">
    <value>Await</value>
  </data>

CS strings:

  <data name="Deprecated" xml:space="preserve">
    <value>deprecated</value>
  </data>
  <data name="Obsolete" xml:space="preserve">
    <value>Obsolete</value>
  </data>
  <data name="Extension" xml:space="preserve">
    <value>extension</value>
  </data>
  <data name="Awaitable" xml:space="preserve">
    <value>awaitable</value>
  </data>
  <data name="AwaitableExtension" xml:space="preserve">
    <value>awaitable, extension</value>
  </data>
  <data name="Await" xml:space="preserve">
    <value>await</value>
  </data>
  <data name="OrganizeUsings" xml:space="preserve">
    <value>Organize Usings</value>
  </data>

How should this be handled?